stdlib/utils/is.lua

```diff
@@ -51,10 +51,6 @@ local M = {}
 local type = type
 local abs = math.abs
 
-local function _find(key)
```

Comment:
Are you using the factorio .luacheckrc?  unused warnings that start or end with _ are ignored